packages/cmcd-validator-library/src/tests/testCases/headerTestCases.js

```diff
@@ -218,7 +218,7 @@ export const headerTestCases = [
         {
           type: 'unknown-key',
           key: '',
-          value: undefined,
+          value: true,
           description: "The key '' is not reserved.",
         },
```

Comment:
I dont uderstand why we are throwing this error.

Reply:
Well, me nether,the errors on that test are not related to the fixes in this ticket. Our changes impact on the value True, but don't make any changes on throwing that error. I will be happy of adding a new ticket to see that if you wish.